README.md

```diff
@@ -65,3 +65,22 @@ If you are just browsing this repository on GitHub, you may not see some of the
 ## Behat tests
 
 So that CircleCI will have some test to run, this repository includes a configuration of [WordHat](https://wordhat.info/), A WordPress Behat extension. You can add your own `.feature` files within `/tests/behat/features`. [A fuller guide on WordPress testing with Behat is forthcoming.](https://github.com/pantheon-systems/documentation/issues/2469)
+
+## Working locally with Lando
+To get started using Lando to develop locally complete these one-time steps. Please note than Lando is an independent product and is not supported by Pantheon. For further assistance please refer to the [Lando documentation](https://docs.devwithlando.io/).
+
+* [Install Lando](https://docs.devwithlando.io/installation/installing.html), if not already installed.
+* Clone this repository locally.
+* Run `lando init` and follow the prompts, choosing the Pantheon recipe followed by entering a valid machine token and selecting the Pantheon site created by [the Terminus build tools plugin].(https://github.com/pantheon-systems/terminus-build-tools-plugin).
+* Run `lando start` to start Lando.
+    - Save the local site URL. It should be similar to `https://<PROJECT_NAME>.lndo.site`.
+* Run `lando terminus auth:login --machine-token=<TOKEN>` to authenticate Terminus inside of Lando.
```

Comment:
Is this step necessary? During `lando init` I was asked `Choose a Pantheon account`. I think that step served the same purpose.

Reply:
I had to do it the first time I tried to do `lando terminus` to run commands. Does that work for you without authorizing? Maybe it not being logged in for me was a fluke.

Reply:
Okay, I am good with removing it. Doing so now